src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(action, num) {
+      this.result = action(this.result, num);
```

Comment:
pass just value from operate

```suggestion
    operate(action, num) {
      action(num);
```

Reply:
Если это писать в методах add и тд, оно не работает, ищет this.result в operate.

Reply:
add, subtract and others are also object's methods and they have access to this